README.md

```diff
@@ -26,9 +26,7 @@ See the example folder for an executable example.
 
 var Hapi = require('hapi'),
```

Comment:
Even if we don't want to update the rest of the code I think we should update the readme to use the new ES6 syntax because this is a public project